packages/vite/src/index.ts

```diff
@@ -29,6 +28,45 @@ export default function redwoodPluginVite(): PluginOption[] {
   const relativeEntryPath = path.relative(rwPaths.web.src, clientEntryPath)
 
   return [
+    {
```

Comment:
Moved this plugin up here because it has `order: 'pre'` which will make it run firsts. So intuitively it makes sense to me to have it first in the array of plugins. The reason it was further down before was because we wanted to group it with the other env-related plugin. But now that we removed that one that argument doesn't hold true anymore